tests/Doctrine/Tests/ORM/Functional/Ticket/GH6531Test.php

```diff
@@ -0,0 +1,242 @@
+<?php
+
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * @group GH-6531
+ */
+class GH6531Test extends \Doctrine\Tests\OrmFunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setup();
+
+        try {
+            $this->_schemaTool->createSchema(array(
+                $this->_em->getClassMetadata(__NAMESPACE__ . '\GH6531User'),
+                $this->_em->getClassMetadata(__NAMESPACE__ . '\GH6531Address'),
+                $this->_em->getClassMetadata(__NAMESPACE__ . '\GH6531Article'),
+                $this->_em->getClassMetadata(__NAMESPACE__ . '\GH6531ArticleAttribute'),
+                $this->_em->getClassMetadata(__NAMESPACE__ . '\GH6531Customer'),
+                $this->_em->getClassMetadata(__NAMESPACE__ . '\GH6531Order'),
+                $this->_em->getClassMetadata(__NAMESPACE__ . '\GH6531OrderItem'),
+                $this->_em->getClassMetadata(__NAMESPACE__ . '\GH6531Product'),
+            ));
+        } catch (\Exception $e) {
```

Comment:
Catch only tools exceptions, please

Reply:
I've copy-pasted the try/catch from some older test. Is it actually good for something? It's never used in more recent tests as I can see now. Shouldn't I rather throw it away?